servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/publish/jmx/KarafJMXPublisher.java

```diff
@@ -142,21 +142,23 @@ public int publish(IServer server, IModule[] module) {
 	 * @see org.fusesource.ide.server.karaf.core.publish.IPublishBehaviour#uninstall(org.eclipse.wst.server.core.IServer, org.eclipse.wst.server.core.IModule)
 	 */
 	@Override
-	public boolean uninstall(IServer server, IModule[] module) {
+	public boolean uninstall(IServer server, IModule[] module, String symbolicName, String version) {
 		if (this.jmxc == null) connect(server);
 		boolean unpublished = false;
 		try {
 			// insert a project refresh here
 			IProject project = module[0].getProject();
 			project.refreshLocal(IProject.DEPTH_INFINITE, new NullProgressMonitor());
 			
-			String version = KarafUtils.getBundleVersion(module[0], null);
+//			String version = KarafUtils.getBundleVersion(module[0], null);
+//			String version2 = KarafUtils.getBundleVersion(module[0], null);
```

Comment:
Do we still need this commented code? Seem sthat now the version is provided as a parameter